### ISSUE TITLE
Won't start since newest update

### ISSUE THREAD

**Issue by chris41g** [SickChill/SickChill#7178]
### Before submitting your issue:

Enable debug logging in SickChill settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: master/ed50be8a298d009576ff7b70e07abbea40817350
OS: ubuntu 21.04 hirsute
Browser: firefox 89.0
What you did: updated to newest commits
What happened: sickchill failed to start
What you expected: for sickchill to start
Logs:
```
/opt/sickchill$ sudo -u sickchill python /opt/sickchill/SickChill.py -q --daemon --nolaunch --pidfile=/opt/sickchill/sickchill.pid --datadir=/opt/sickchill
Traceback (most recent call last):
  File "/opt/sickchill/SickChill.py", line 14, in <module>
    import sickchill.start
  File "/opt/sickchill/sickchill/__init__.py", line 1, in <module>
    from .show.indexers import indexer, ShowIndexer
  File "/opt/sickchill/sickchill/show/indexers/__init__.py", line 1, in <module>
    from .handler import ShowIndexer
  File "/opt/sickchill/sickchill/show/indexers/handler.py", line 3, in <module>
    from sickchill import logger, settings
  File "/opt/sickchill/sickchill/logger.py", line 72
    msg = re.sub(fr"\b({item})\b", "*" * 8, msg)
                                ^
SyntaxError: invalid syntax

```

**Comment by Xx-Ness-xX** [comment]
I have the exact same issue except I am running Windows 10

**Comment by kyuuk** [comment]
Duplicate of #7177

**Comment by chris41g** [comment]
> 
> 
> Duplicate of #7177

I don't think it is a duplicate. They are both syntax errors yes, but in different files and the proposed temp fix from #7177 doesn't fix my issue

**Comment by chris41g** [comment]
Closing this because apparently I am slow. I recently did a distro upgrade and it made the default python python2.

**Comment by kyuuk** [comment]
yes

> 
> 
> > Duplicate of #7177
> 
> I don't think it is a duplicate. They are both syntax errors yes, but in different files and the proposed temp fix from #7177 doesn't fix my issue

yes my bad mixed with another issue
